demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -385,6 +385,13 @@ def is_valid_default_argument(self):
 
         for command in commands:
             default_args = set()
+            if command.get('arguments', []) is None:
```

Comment:
what happens what `arguments` is not in `command`?

Reply:
same as happened before - `command.get('arguments', [])` will return [], which is not None, so the `if` statement will be False and we continue to the old flow which iterates over the arguments list (empty list so it won't really iterate over anything).